package.json

```diff
@@ -21,6 +21,7 @@
     "glob": "^5.0.14",
     "iferr": "^0.1.5",
     "mkdirp": "^0.5.1",
+    "options": "0.0.6",
```

Comment:
Carrot versions please. Is this module stable? What's the reason it's got such a low version number?